moroScript.jsx

```diff
@@ -115,6 +117,23 @@
         return rtnWord;
     }
 
+    //Process dict with count to sorted dict without count value
+    function sortAndRemoveCount(dict) {
+        var toRtn = JSON.parse(JSON.stringify(dict));
+        for(var i=0; i<toRtn.length; ++i) {
+            toRtn[i]["moroword"].sort(function(a, b) {
+                return parseFloat(b["count"]) - parseFloat(a["count"]);
+            });
+            var moroWordsArray = []
+            for (var j=0; j<toRtn[i]["moroword"].length; ++j) {
```

Comment:
We decided beforehand not to change the rendering code. My reactJS is also very weak. It would be great if you could show what it would look like changing the rendering code. I would also bring this up in the next meeting.